src/OVAL/CMakeLists.txt

```diff
@@ -64,6 +64,10 @@ if (ENABLE_PROBES)
 	"oval_probe_meta.h"
 	"probes/probe-api.c"
 	"probes/_probe-api.h"
+	"probes/fsdev.c"
```

Comment:
This is a badly resolved merge. Those files are already moved to lines 74-78, they shouldn't be duplicated here.